Stripe/StripeiOSTests/FormSpecProviderTest.swift

```diff
@@ -384,10 +384,10 @@ class FormSpecProviderTest: XCTestCase {
 
         // Validate that we were not able to override the spec read in from disk
         let epsUpdated = try XCTUnwrap(sut.formSpec(for: paymentMethodType))
-        XCTAssertEqual(epsUpdated.fields.count, 2)
+        XCTAssertEqual(eps.fields.count, 5)
         XCTAssertEqual(
-            epsUpdated.fields.first,
-            .name(FormSpec.NameFieldSpec(apiPath: nil, translationId: nil))
+            eps.fields.first,
+            .name(FormSpec.NameFieldSpec(apiPath: ["v1": "billing_details[name]"], translationId: nil))
```

Comment:
These need to reference `epsUpdated`, not the original `eps` variable.

Reply:
Oops, copy-pasta